examples/radix-example-vite-spa-env-variables/src/environmentVariables.ts

```diff
@@ -0,0 +1,11 @@
+const envVars = {
+  deploymentEnvironment: "${VITE_DEPLOYMENT_ENVIRONMENT}",
+};
+  
+export function envVariables() {
+  return {
+    deploymentEnvironment: !envVars.deploymentEnvironment.includes("{VITE_DEPLOYMENT_ENVIRONMENT}")
+      ? envVars.deploymentEnvironment
+      : import.meta.env.VITE_DEPLOYMENT_ENVIRONMENT,
```

Comment:
Could we write something here about how it works?

```suggestion
    // `envsubst` should modify this variable, but double check and fallback to build environment if not
    deploymentEnvironment: !envVars.deploymentEnvironment.includes("{VITE_DEPLOYMENT_ENVIRONMENT}")
      ? envVars.deploymentEnvironment
      : import.meta.env.VITE_DEPLOYMENT_ENVIRONMENT,
```

Reply:
Added a commentstring, have a look if it looks ok to you!